docs/dictionary.html

```diff
@@ -7939,6 +7941,25 @@
       </div>
     <div class="dict_entry">
       <h3>
+        <a name="stop-inspecting" id="stop-inspecting">stop-inspecting</a>
+      </h3>
+      <h4>
+        stop-inspecting <i>agent</i>
+      </h4>
+      <p>
+        Closes the agent monitor for the given agent (turtle or patch).
+        In the case that no agent monitor is open, stop-inspecting does
+        nothing.
+      <pre>
+stop-inspecting patch 2 4
+;; the agent monitor for that patch closes
+stop-inspecting one-of sheep
```

Comment:
Not sure I have a better suggestion from the top of my head, but that's a very unlikely use case...